.gitignore

```diff
@@ -23,9 +23,11 @@ dmypy.json
 
 # IDEs
 .idea
-*.swp
 .vscode
 
+# vim temporary files
+.*.sw?
```

Comment:
As I understand it, this will only work for vim swap files where the file starts with a dot, because in git ignore syntax, `*` need not be preceded by a `.`.

Reply:
Yes. You are absolutely right. What my change actually does is ignore ```.tmp.sw?``` files. I should specify that in the PR description. Also, I'll add the correct syntax to ignore ```.sw?``` files. Thanks.

Reply:
Did some digging and according to https://github.com/github/gitignore/blob/master/Global/Vim.gitignore, these changes should do the trick recursively.